tests/test_menu.py

```diff
@@ -1,112 +1,160 @@
-from tests.mocks import MockConsole, MockConsoleIO
+import pytest
+
+from tests.mocks import MockConsole, MockConsoleIO, MockPersisterIO
 from ttt import constants
 from ttt.console_ui.console import Console
 from ttt.console_ui.menu import Menu
 from ttt.messages import play_again_message, end_game_message
+from ttt.persister.persister import Persister
 from ttt.players.human_player import HumanPlayer
 from ttt.players.simple_computer_player import SimpleComputerPlayer
 
 
+@pytest.fixture
+def json_data():
+    return '{"1": {"board": ["X", "O", "X", "O", "5", "6", "7", "8", "9"], ' \
```

Comment:
If this is meant to be a string rather than a dict, have you considered the multiline quotes (`""" ...text... """`)?

Why not something like:

```
my_dict = { 'dict': 'stuff' }
return json.dumps(my_dict)
```

Reply:
This makes sense to me but is it still  the right approach if I've moved away from using the fixture?

I supposed I could have two constants:

SAVED_GAME_DATA_DICT = {'abc': '123'}
SAVED_GAMES_JSON_DATA = json.dumps(SAVED_GAME_DATA_DICT)

Reply:
Actually disregard that - based on your subsequent comment I'm going to do some shifting around here anyway.